src/tests/unit/tests/DetailsView/components/details-view-command-bar.test.tsx

```diff
@@ -59,8 +58,8 @@ describe('DetailsViewCommandBar', () => {
                     title: 'test title',
                 } as Assessment;
             });
-        reportGeneratorMock = Mock.ofType<ReportGenerator>(undefined, MockBehavior.Strict);
-        reportGeneratorProviderMock = Mock.ofType<ReportGeneratorProvider>(undefined, MockBehavior.Strict);
+        reportGeneratorMock = Mock.ofType<ReportGenerator>(undefined, MockBehavior.Loose);
+        reportGeneratorProviderMock = Mock.ofType<ReportGeneratorProvider>(undefined, MockBehavior.Loose);
```

Comment:
it looks like this reportGeneratorProviderMock doesn't need to be `Loose`, it's being `setup` on the next line